src/main.zig

```diff
@@ -248,7 +251,7 @@ pub const ObjectManager = struct {
                 }
             }
 
-            try te.draw();
+            try te.drawText();
```

Comment:
why is this called "drawText"? What else can it draw?

Reply:
The background/border.  I think with your help we've finally got a decent split.  The background/border is drawn in install(), then events are processed, then the text is drawn (and also the cursor and focus outline).  Hmm maybe that name is bad, maybe draw() is best.  Thoughts?

Reply:
well that seems complicated. is the graphic appended to a drawlist or drawn directly?

Reply:
Yes it is complicated!  Right now a drawlist, then everything is drawn in Window.end().  I think that for the main window we want to draw them directly, so that a user can draw custom stuff into a widget after dvui has drawn the background/border, but still use a drawlist for floating windows.  But nobody has asked for that yet.

Reply:
then just `draw()` is best